source/handlers/desktopparser.d

```diff
@@ -187,11 +187,18 @@ Component parseDesktopFile (GeneratorResult gres, string fname, string data, boo
         if (key.startsWith ("Name")) {
             auto val = getValue (df, key);
             checkDesktopString (key, val);
-            cpt.setName (val, locale);
+            /* run backend specific hooks */
+            auto translations = gres.pkg.processDesktopFile (df, val);
```

Comment:
I was worried shortly here, but `null` and an empty array are treated equally by D, so this is fine :)